tests/form-builder-tests-unit.csproj

```diff
@@ -9,6 +9,7 @@
 
   <ItemGroup>
     <PackageReference Include="AWSSDK.S3" Version="3.3.104.41" />
+    <PackageReference Include="AWSSDK.SimpleEmail" Version="3.3.101.175" />
```

Comment:
is this package needed in unit test project or can be removed.

Reply:
It is needed, the tests use the SimpleEmail.Model